test/gen-attributions_test.go

```diff
@@ -0,0 +1,101 @@
+package test
```

Comment:
Ideally if this is a `*test` package it should be in `/cmd` but we could address that in the refactoring patch